app/services/parse_website_for_data.rb

```diff
@@ -0,0 +1,80 @@
+class ParseWebsiteForData
+  WEBSITE = "https://www.greenvillesc.gov".freeze
```

Comment:
Change this line if you need to replace the web-site (if the same web-site will change it's domain)